distage/distage-framework-docker/src/test/scala/izumi/distage/testkit/docker/DistageTestDockerBIO.scala

```diff
@@ -71,3 +71,15 @@ final class DistageTestDockerBIOSecondEnv extends DistageTestDockerBIO {
 final class DistageTestDockerBIOThirdEnv extends DistageTestDockerBIO {
   override protected def config: TestConfig = super.config.copy(logLevel = Log.Level.Error)
 }
+
+// Labels tests
+final class DistageTestDockerTagsTest1 extends DistageTestDockerBIO {
```

Comment:
@VladPodilnyk 
These tests only test launching the container, they're not testing if the tags are there - you may probably want to test that.

Reply:
Yup, thx!
I intended to do so, but I've got an issue with those test and I've hopped in to find a cause hence it's not fully complete.

Reply:
I've fixed tests, so they check if tag exist, but it doesn't check a situation where two containers with same image are available, but DIstage choose that one with tags